Library/Formula/saldl.rb

```diff
@@ -0,0 +1,47 @@
+class Saldl < Formula
+  desc "CLI downloader optimized for speed and early preview."
+  homepage "https://saldl.github.io"
+  url "https://github.com/saldl/saldl/archive/v31.tar.gz"
+  sha256 "146a980ae2109a391c7a8ab6a1c525458db9bf18f3f0477731c9b876630078b4"
+
+  head do
+    # git describe does not work with shallow clones
+    url "https://github.com/saldl/saldl.git", :shallow => false
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "asciidoc" => :build
+  # Install DTDs locally for a faster and more reliable build result
+  depends_on "docbook-xsl" => :build
+
+  depends_on "libevent"
+
+  # Add option to use keg_only curl formula instead of the version provided by OSX
```

Comment:
I would prefer it if this option stays, unless you consider it a blocker.

1- I would like to provide instructions that don't require manually editing formulas for HTTP/2 support.

2- Setting a default protocol is a library feature that appeared in `7.45`. The default protocol is set to `HTTPS` if the required `libcurl` version is available.

3- OSX will always lag behind upstream releases. And each release comes with a set of fixes and features that I will probably make use of. `HTTPS proxy` support is an important feature that is expected to be introduced in the `7.47` release.